src/ui/public/vislib/visualizations/_map.js

```diff
@@ -47,16 +51,15 @@ export default function MapFactory(Private) {
     this._valueFormatter = params.valueFormatter || _.identity;
     this._tooltipFormatter = params.tooltipFormatter || _.identity;
     this._geoJson = _.get(this._chartData, 'geoJson');
-    this._mapZoom = params.zoom || defaultMapZoom;
+    this._mapZoom =  Math.max(Math.min(params.zoom || defaultMapZoom, tilemapOptions.maxZoom), tilemapOptions.minZoom);
```

Comment:
In 4.5 the saved zoom was getting pulled out of the geoJson object. Did we find out params is the correct place to get it?